src/Angor/Client/Pages/Create.razor

```diff
@@ -847,6 +849,21 @@
 
         try
         {
+            // Check if the FounderKey is in use and get the total keys used
+            var keyCheckResult = await FounderKeyService.CheckFounderKeyAsync(project.ProjectInfo.FounderKey);
+
+            if (keyCheckResult.IsKeyInUse)
+            {
+                notificationComponent.ShowErrorMessage("The FounderKey is already in use. Please use a different key.");
```

Comment:
In this case we need to "fix" the data in the local storage because the user can't select a founder key to use, perhaps add a message to rescan or wipe the storage and recover the wallet again.